apis/core/v1alpha2/function_conversion.go

```diff
@@ -82,6 +82,15 @@ func (src *Function) ConvertTo(dstRaw conversion.Hub) error {
 		}
 	}
 
+	if src.Status.URL != "" {
+		addressType := v1beta1.ExternalAddressType
```

Comment:
The original URL should be an internal address I think, it can only be access from inside a pod
@wanjunlei @tpiperatgod , what do you think?